site/content/docs/0.0/customize/custom-libraries.md

```diff
@@ -59,3 +59,40 @@ In the absence of a designer, it’s advisable to reach out to the [OUDS Web cor
 ### Building and deploying documentation
 
 You are responsible for building and deploying your documentation. You can start with our existing documentation, modifying or removing sections as needed to fit your new design. Alternatively, you may choose to build your documentation from scratch using any preferred framework.
+
+### Custom color modes
+
+<!-- TODO LM: Do we Ouds ify this paragraph or remove it ? -->
```

Comment:
Hmm good question here,
I'm not completely sure on how we will manage Sosh, but I think most of brand/countries wont have to declare custom color mode. Color mode correspond pretty much to what is done in Figma with tokens. A new brand will have is own instance of tokens but will not be a color mode in OUDS Web. So I think this part should not be useful but I'm not completely sure :)